vmm/src/vm.rs

```diff
@@ -442,11 +446,11 @@ impl Vm {
     ) -> Result<Self> {
         #[cfg(target_arch = "x86_64")]
         hypervisor.check_required_extensions().unwrap();
-        let vm = hypervisor.create_vm().unwrap();
+        let vm = hypervisor.create_vm(None).unwrap();
```

Comment:
At the time of creating Vm object in hypervisor, the Vm object in vmm is not fully formed. This forced me to declare vmmops in KvmVM to be Option<Arc<dyn vm::VmmOps>>. This way I set vmmops to **None** first and eventually populate it with correct information.

Does this seem reasonable? Any better way to handle this?

Reply:
Does it need to be inlcluded as a member variable in the hypervisor:vm. Can't we just pass the vmmops to the hypervisor functions that need this?

Reply:
Both these seem reasonable to me.